docs/docs/basics/01-tx-lifecycle.md

```diff
@@ -157,37 +157,41 @@ must be in this proposer's mempool.
 ## State Changes
 
 The next step of consensus is to execute the transactions to fully validate them. All full-nodes
-that receive a block proposal from the correct proposer execute the transactions by calling the ABCI functions
-[`BeginBlock`](./00-app-anatomy.md#beginblocker-and-endblocker), `DeliverTx` for each transaction,
-and [`EndBlock`](./00-app-anatomy.md#beginblocker-and-endblocker). While each full-node runs everything
-locally, this process yields a single, unambiguous result, since the messages' state transitions are deterministic and transactions are
+that receive a block proposal from the correct proposer execute the transactions by calling the ABCI function `FinalizeBlock`. 
```

Comment:
Possibly here, still link the `./00-app-anatomy.md#beginblocker-and-endblocker` section which will probably explain what FinalizeBlock does once updated.